samples/doc_writer_sample_output.py

```diff
@@ -347,8 +348,12 @@
         },
         {
             "@id": "vocab:dummyClassCollection",
-            "@type": "hydra:Class",
+            "@type": "Collection",
```

Comment:
@chrizandr Do we want `hydra:Class` here or `Collection` or `DummyClassCollection?` Going by the default spec, it seems like DummyClassCollection will be a good choice, but in the spec repo, i have seen using 'Collection' only as a type.